appendices/versions.txt

```diff
@@ -188,37 +188,71 @@ number of key extensions into the core API:
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_KHR_sampler_ycbcr_conversion`
 
 If the `<<VK_KHR_sampler_ycbcr_conversion>>` extension is not supported,
-support for the <<features-samplerYcbcrConversion,samplerYcbcrConversion>>
+support for the <<features-samplerYcbcrConversion, pname:samplerYcbcrConversion>>
 feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceSamplerYcbcrConversionFeatures::pname:samplerYcbcrConversion
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
-=== Differences relative to `VK_KHR_16bit_storage`
+=== Differences relative to `VK_KHR_shader_draw_parameters`
 
-If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
-the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
-is optional.
+If the `<<VK_KHR_shader_draw_parameters>>` extension is not supported,
+support for the <<features-shaderDrawParameters, pname:shaderDrawParameters>>
+feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceShaderDrawParametersFeatures::pname:shaderDrawParameters
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:shaderDrawParameters
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
+
+=== Differences relative to `VK_KHR_variable_pointers`
+
+If the `<<VK_KHR_variable_pointers>>` extension is not supported,
+support for the <<features-variablePointersStorageBuffer,
+pname:variablePointersStorageBuffer>> feature is optional.
+Support for this feature is defined by
+slink:VkPhysicalDeviceVariablePointersFeatures::pname:variablePointersStorageBuffer
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:variablePointersStorageBuffer
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
+
+=== Additional Vulkan 1.1 Feature Support
+
+[[versions-1.1-new-features]]
+In addition to the promoted extensions described above, Vulkan 1.1 added
+support for:
+
+  * The <<shaders-group-operations, group operations>> and
+    <<shaders-scope-subgroup, subgroup scope>>.
+  * The <<memory-protected-memory, protected memory>> feature.
+  * A new command to enumerate the instance version:
+    flink:vkEnumerateInstanceVersion.
+  * The <<features-shaderDrawParameters, pname:shaderDrawParameters>>
+    feature query struct, where `<<VK_KHR_shader_draw_parameters>>` did have
+    none.
```

Comment:
There are some massive Swapchain changes that I should also note here.

Reply:
Swapchains are in core so that doesn't seem like it should be added here.

Reply:
Well, the 1.1 modifies the pre-existing extension. I am on the fence. If one is being pedantic, then maybe it does not belong here. On the other hand for the sake of completeness, it feels like it should.

Reply:
please do it in a separate MR, regardless. This one has already grown way beyond it's scope.

Reply:
Yup